### PR TITLE
functions update

### PR DESCRIPTION
**Pull request by double-y**
* make value column argument optional in kv functions
* add score function to replace augment

### CODE REVIEW

File: R/util.R [hunk 1]
```diff
@@ -25,20 +25,32 @@ simple_cast <- function(data, row, col, val, fun.aggregate=mean, fill=0){
...
+    mat[mat == 0] <- fill
+    mat
+  }else{
+    data %>%  reshape2::acast(fml, value.var=val, fun.aggregate=fun.aggregate, fill=fill)
+  }
 }
 
 #' Cast data to sparse matrix by choosing row and column from a data frame
-sparse_cast <- function(data, row, col, val=NULL, fun.aggregate=sum){
+sparse_cast <- function(data, row, col, val=NULL, fun.aggregate=sum, count = FALSE){
```
Comment (hideaki) [inline R/util.R]: Please add some comment on what the count argument does.

File: R/broom_wrapper.R [hunk 2]
```diff
@@ -71,3 +71,18 @@ augment_kmeans <- function(df, model, data){
     }
   })
 }
+
+#' augment wrapper
+#' @export
+score <- function(df, model, ...){
```
Comment (hideaki) [inline R/broom_wrapper.R]: We will be calling this one predict.

File: R/util.R (hunk 1 above)
Comment (double-y) [inline R/util.R]: I put a comment

File: R/broom_wrapper.R (hunk 2 above)
Comment (double-y) [inline R/broom_wrapper.R]: Changed to predict

**Pull request merged**